### PR TITLE
[WB-5896] remove block on github urls

### PR DESCRIPTION
**Pull request by stephchen**
https://wandb.atlassian.net/browse/WB-5896

Description
-----------

remove block on/add test for github url support, fix lack of default entrypoint from the last refactor, improve messaging a little

Testing
-------

added test

### CODE REVIEW

File: tests/launch/test_launch_cli.py [hunk 1]
```diff
@@ -136,3 +134,27 @@ def test_launch_no_docker_exec(
...
+
+    with runner.isolated_filesystem():
+        # result = runner.invoke(
+        #     cli.launch,
+        #     [
+        #         '-c,',
+        #         json.dumps(config),
+        #         'https://github.com/wandb/examples',
+        #     ]
+        # )
```
Comment (KyleGoyette) [inline tests/launch/test_launch_cli.py:155]: old code comment

File: wandb/sdk/launch/utils.py [hunk 2]
```diff
@@ -114,14 +126,19 @@ def create_project_from_spec(run_spec, api):
     else:
         name = "{}_{}_launch".format(project, run_id)  # default naming scheme
 
+    overrides = run_spec.get("overrides", {})
+    if "entry_point" not in overrides:
+        wandb.termlog("Entry point for project not specified, defaulting to main.py")
```
Comment (KyleGoyette) [inline wandb/sdk/launch/utils.py:131]: I'm pretty sure we default to whatever was used to run the original script.

**Review by KyleGoyette — COMMENTED** [review]
One note on messaging, and some commented code to remove

File: wandb/sdk/launch/utils.py (hunk 2 above)
Comment (stephchen) [reply wandb/sdk/launch/utils.py:131]: ah you're right we handle that elsewhere, fixing

**Review by KyleGoyette — APPROVED** [review]
nice!

**Pull request merged**